### ISSUE TITLE
Can I combine two properties under "settings" ? 

### ISSUE THREAD

**Issue by gkatsanos** [benmosher/eslint-plugin-import#1462]
This is my configuration: 
```json
{
  "plugins": [
    "vue",
    "prettier"
  ],
  "extends": [
    "plugin:vue/recommended",
    "airbnb-base"
  ],
  "env": {
    "jest": true
  },
  "parserOptions": {
    "sourceType": "module",
    "ecmaVersion": 2018
  },
  "globals": {
    "App": true
  },
  "settings": {
    "import/extensions": [
      ".vue"
    ],
    "import/resolver": {
      "node": {
        "paths": ["app/javascript"]
      }
    }
  },
}
```

It seems that I cannot combine `import/extensions` and `import/resolver` ? 
I want to be able to import .vue files (modules?) without having to write .vue in the import statement and inform eslint-plugin-import about it so it doesn't throw a linting error.

This is an example file I use:

```js
// app/javascript/tenant_group_admin/routes.js

import tenantsNew from 'tenant_group_admin/components/tenants/new';
import tenantsShow from 'tenant_group_admin/components/tenants/show';
import gatewaysIndex from 'tenant_group_admin/components/gateways/index';

export default [
  { path: '/tenants/:id', name: 'tenants/show', component: tenantsShow },
  { path: '/tenant-groups/:tenantGroupId/tenants/new', name: 'tenants/new', component: tenantsNew },
  { path: '/tenant-groups/:tenantGroupId/gateways', name: 'gateways/index', component: gatewaysIndex },
];
```

and a location of one of the imports:
```
app/javascript/tenant_group_admin/components/tenants/new.vue
```

```
║ Line     │ Column   │ Type     │ Message                                                │ Rule ID              ║
╟──────────┼──────────┼──────────┼────────────────────────────────────────────────────────┼──────────────────────╢
║ 1        │ 24       │ error    │ Unable to resolve path to module                       │ import/no-unresolved ║
║          │          │          │ 'tenant_group_admin/components/tenants/new'.           │                      ║
║ 2        │ 25       │ error    │ Unable to resolve path to module                       │ import/no-unresolved ║
║          │          │          │ 'tenant_group_admin/components/tenants/show'.          │                      ║
║ 3        │ 27       │ error    │ Unable to resolve path to module                       │ import/no-unresolved ║
║          │          │          │ 'tenant_group_admin/components/gateways/index'.        │                      ║
```
*PS I use https://github.com/rails/webpacker so I can't feed the webpack config to the resolver - it lives somewhere in node_modules*

**Comment by gkatsanos** [comment]
@ljharb so my configuration as I pasted it will not work? I *need* the webpack configuration?
Can't I just declare myself a path alias and a file extension to look for?  (I find hard to believe)

**Comment by gkatsanos** [comment]
Hmm, what about using a plug-in? There's a vue eslint one..

**Comment by gkatsanos** [comment]
I did but I admit the docs of the structure of the object of import-resolver are a bit cryptic. (as always, some examples would go very far)

**Comment by gkatsanos** [comment]
Saw that. But how do I combine it with path?

**Comment by gkatsanos** [comment]
> ```
> "settings": {
>     "import/extensions": [
>       ".vue"
>     ],
>     "import/resolver": {
>       "node": {
>         "paths": ["app/javascript"],
>         "extensions": [".js", ".vue"]
>       }
>     }
>   },
> ```
> 
> ?

that worked. Thank you!
